packages/nocodb-sdk/src/lib/error/nc-base.error.ts

```
@@ -0,0 +1,96 @@
+import { NcErrorType } from '~/lib/globals';
+
+export class NcBaseError extends Error {
+  constructor(message: string) {
+    super(message);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)